services/github/github-go-mod.tester.js

```diff
@@ -15,7 +15,7 @@ t.create('Go version (from branch)')
     message: isVPlusDottedVersionAtLeastOne,
   })
 
-t.create('Go version (mongorepo)')
+t.create('Go version (monorepo)')
```

Comment:
:rofl: good catch!

Reply:
My but was telling me it's about Mongo 🤣